PoSSapi/src/Application/Businesses/Validators/UpdateBusinessCommandValidator.cs

```diff
@@ -0,0 +1,15 @@
+using FluentValidation;
+using PoSSapi.Application.Businesses.Commands;
+
+namespace PoSSapi.Application.Businesses.Validators;
+
+public class UpdateBusinessCommandValidator : AbstractValidator<UpdateBusinessCommand>
+{
+    public UpdateBusinessCommandValidator()
+    {
+        RuleFor(v => v.Name)
+            .NotEmpty()
+            .NotNull()
```

Comment:
Nits: since name is not a nullable property, I doubt that notNull check does anything